exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Service.java

```diff
@@ -53,6 +53,24 @@ default void initialize(Fork fork, Configuration configuration) {
     // No configuration
   }
 
+  /**
+   * Performs resuming of previously stopped service instance. This method is called <em>once</em>
+   * after restarting a service instance.
+   *
+   * <p><em>Note that changing the data during resume operations is not allowed
+   * due to violation migration procedure.</em>
+   *
+   * @param configuration the service configuration parameters
+   * @throws ExecutionException if the configuration parameters are not valid (e.g.,
+   *     malformed, or do not meet the preconditions). Exonum will save the error into
+   *     {@linkplain com.exonum.binding.core.blockchain.Blockchain#getCallErrors(long)
+   *     the registry of call errors} if the resuming fails
+   * @see Configurable
```

Comment:
Configurable is not related to this, but initialize (if we decide to keep the method at all)